src/pages/software-engineering/version-control-system/index.md

```diff
@@ -9,7 +9,16 @@ This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/
 
 <!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
 
+A component of software configuration management, version control, also known as revision control or source control, is the management of changes to documents, computer programs, large web sites, and other collections of information. Changes are usually identified by a number or letter code, termed the "revision number", "revision level", or simply "revision". For example, an initial set of files is "revision 1". When the first change is made, the resulting set is "revision 2", and so on. Each revision is associated with a timestamp and the person making the change. Revisions can be compared, restored, and with some types of files, merged.
+
+Version control systems (VCS) most commonly run as stand-alone applications, but revision control is also embedded in various types of software such as word processors and spreadsheets, collaborative web docs and in various content management systems, e.g., Wikipedia's Page history. Revision control allows for the ability to revert a document to a previous revision, which is critical for allowing editors to track each other's edits, correct mistakes, and defend against vandalism and spamming.
+
 #### More Information:
+
+<a href='https://www.atlassian.com/git/tutorials/what-is-version-control' target='_blank' rel='nofollow'>What is version control</a>.
+<a href='http://guides.beanstalkapp.com/version-control/intro-to-version-control.html' target='_blank' rel='nofollow'>Intro to version control</a>.
+<a href='https://betterexplained.com/articles/a-visual-guide-to-version-control/' target='_blank' rel='nofollow'>A visual guide to version control</a>.
+
 <!-- Please add any articles you think might be helpful to read before writing the article -->
 
```

Comment:
<a href = 'https://en.wikipedia.org/wiki/Version_control' target ='_blank' rel='nofollow'>Version Control - Wikipedia</a>
<p><a href='https://git-scm.com/' target='_blank' rel='nofollow'>Git</a> is one of the popular version control systems used today.</p>